### ISSUE TITLE
OneFuzz CLI 6.3.0 failed with: templateERROR:cli:command failed: local variable 'json' referenced before assignment

### ISSUE THREAD

**Issue by stishkin** [microsoft/onefuzz#2877]
This is not 100% reproducible, but it happens. 

```
Onefuzz CLI failed due to: Error  (1, "",   "INFO:onefuzz:creating libfuzzer from templateERROR:cli:command failed: local variable 'json' referenced before assignment",   "template libfuzzer basic --duration 168 --fuzzing_target_options " -runs=10000" --colocate_all_tasks --vm_count 1 --reboot_after_setup --readonly_inputs inputs --notification_config ado_config.json --setup_dir supt_dir --target_exe target_exe --minimized_stack_depth 4 PROJ NAME BUILD POOL")
```

AB#43451101

**Comment by stishkin** [comment]
Might be fixed in 6.4 but still would be nice to know the root cause

**Comment by nharper285** [comment]
For context, 6.3.0 didn't have any of my config changes so this is likely separate. Continuing to investigate root cause.
